extra/zabbix/vsphere-import/zabbix-vsphere-import

```diff
@@ -473,10 +474,23 @@ class ZabbixVSphere(object):
             'hostname': self.options['vsphere']['hostname']
         }
         vpoller_data = self._call_vpoller_task(msg=msg)
-        self.vsphere_vms = [vm['name'] for vm in vpoller_data]
-
-        missing_hosts = set(self.vsphere_vms) - set(zabbix_hosts)
-
+        # regex of all vm names where logging is not desired.                                       
+        # Get hosts options (templates, groups, macros, ignore_vm(optional))
+        host_options = self._get_zabbix_host_options('vsphere_object_vm')
+        #convert vsphere_exclude_list to regex objects)
+        regex_list = []
+        for regex in self.vsphere_exclude_list:
+            regex_list.append(re.compile(regex))
+        vmreal = []
+        for h in vpoller_data:
+        """
+        Remove hosts on the ignore list (ignore_vm key/values in config file.)                                
+        """                                                                                                   
+            if not any (regex.match(h[u'name']) for regex in regex_list):                                         
```

Comment:
Style fix: white space between function call `any` and it's arguments. Please use `any(iterable)` without the white space.